test/e2e/upgrade_test.go

```diff
@@ -282,3 +331,98 @@ func (tc *testContext) waitForCSINodesWithDrivers(nodes []v1.Node) error {
 		return true, nil
 	})
 }
+
+// deployUnhealthyNodesCheckerJob deploys a Job that checks the number of nodes marked as unhealthy and fails if the
+// number of unhealthy nodes exceeds a threshold
+func (tc *testContext) deployUnhealthyNodesCheckerJob() (*batchv1.Job, error) {
+	os := &v1.PodOS{Name: v1.Linux}
+	selector := map[string]string{"kubernetes.io/os": "linux"}
+	runAsNonRootTrue := true
+	checkerCommand := `
+		#!/bin/bash
+		set -euo pipefail
+		# set working directory
+		cd /tmp
+		# download oc
+		curl -sL https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest/openshift-client-linux.tar.gz -o oc.tar.gz && tar xzvf oc.tar.gz &>/dev/null
```

Comment:
I am a nack on this. At the minimum we should be using an image that has oc.

Reply:
Agree; just need to find the right one that works both in CI and with local clusters.

Reply:
Any recommendations?

Reply:
Hopefully this won't be required as I really don't see a need for a Job here.